core/primitives/src/runtime/migration_data.rs

```diff
@@ -0,0 +1,39 @@
+#[cfg(feature = "protocol_feature_fix_storage_usage")]
+use crate::types::AccountId;
+use std::fmt;
+use std::fmt::{Debug, Formatter};
+use std::sync::Arc;
+
+pub struct MigrationData {
+    #[cfg(feature = "protocol_feature_fix_storage_usage")]
+    pub storage_usage_delta: Vec<(AccountId, u64)>,
+}
+
+impl MigrationData {
+    pub const EMPTY: MigrationData = MigrationData {
+        #[cfg(feature = "protocol_feature_fix_storage_usage")]
+        storage_usage_delta: Vec::new(),
+    };
+}
```

Comment:
Seems like you can use derive(Default) here?

Reply:
Not working for const (although I used it for MigrationContext)